apps/marsha/templates/app/dc.yml.j2

```diff
@@ -61,19 +61,15 @@ spec:
               mountPath: /data/media
             - name: marsha-v-static
               mountPath: /data/static
-{% if MARSHA_VAULT.DJANGO_CLOUDFRONT_PRIVATE_KEY is defined %}
             - mountPath: "{{ marsha_cloudfront_private_key_path | dirname }}"
               name: marsha-cloudfront-private-key-secret
-{% endif %}
       volumes:
         - name: marsha-v-media
           persistentVolumeClaim:
             claimName: marsha-pvc-media
         - name: marsha-v-static
           persistentVolumeClaim:
             claimName: marsha-pvc-static
-{% if MARSHA_VAULT.DJANGO_CLOUDFRONT_PRIVATE_KEY is defined %}
         - name: marsha-cloudfront-private-key-secret
           secret:
             secretName: "{{ marsha_cloudfront_private_key_secret_name }}"
-{% endif %}
```

Comment:
just curious: do you remove it because in our case we always use it or because it works even if it is not set?

Reply:
actually in Marsha it's not required if you don't activate URL signing in CloudFront. I'm not sure though whether we fully planned this possibility in Arnold.

Reply:
@lunika did. Anyway, remove request signature is not something we would recommend right?

Reply:
No and by default you cannot change this setting in our terraform so cloudfront will be configured to restrict viewer access. You have to change this setting in the cloudwatch interface.

Reply:
so it's a half baked thing in Marsha because I put some Django settings to deactivate it :disappointed:. The flag in cloudfront would be easy to add. Should we create an issue for that?

Reply:
Yes! Both here and in Marha's repository.

Reply:
ok let me do it.

Reply:
I think it's only required for Marsha, I've rolled back this feature in Arnold.